presto-iceberg/src/main/java/com/facebook/presto/iceberg/delete/DeleteFile.java

```diff
@@ -0,0 +1,121 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.facebook.presto.iceberg.delete;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.google.common.collect.ImmutableList;
+import org.apache.iceberg.FileContent;
+import org.apache.iceberg.FileFormat;
+
+import java.nio.ByteBuffer;
+import java.util.List;
+import java.util.Map;
+import java.util.Optional;
+
+import static java.util.Objects.requireNonNull;
+
+public class DeleteFile
+{
+    private final FileContent content;
```

Comment:
Hi @majetideepak, 
The Fields that we are mapping here are being added from the DeleteFile in Iceberg library which has the description of each of the fields. Do we still need the comments here?

Reply:
@agrawalreetika ideally yes. You should add a reference to the library and also add a comment here.

Reply:
@agrawalreetika the Presto coding guidelines do not say anything. Mostly, only the interfaces seem to have comments. Since this is an implementation of the Iceberg ContentFile interface, it's fine to not have comments I guess.